apps/meteor/client/sidebar/footer/voip/index.tsx

```diff
@@ -104,8 +103,7 @@ export const VoipFooter = (): ReactElement | null => {
 			dispatchEvent={dispatchEvent}
 			openedRoomInfo={openedRoomInfo}
 			anonymousText={t('Anonymous')}
-			isEnterprise={isEE === true}
-			children={<SidebarFooterWatermark />}
+			isEnterprise={isEnterprise}
 			options={options}
```

Comment:
I purposely put this watermark there, in my view it makes no sense to remove the watermark, I will ask for an approver from the pm of digital workplace @milton-rucks

Reply:
I don't think we should have this distinction

Reply:
I've checked with Renato and Milton and they're aligned.  This change is approved from Product.  An important note here - the overlay of the watermark is temporary when ringer is displayed.  The watermark is not being removed when VoIP is enabled.